shared/actions/chat.js

```diff
@@ -547,12 +548,26 @@ function * _incomingMessage (action: IncomingMessage): SagaGenerator<any, any> {
               })
             }
           }
+
+          let messages = []
+          if (message.type === 'Attachment') {
+            const existingAttachment = yield select(_messageSelector, conversationIDKey, message.messageID)
```

Comment:
:sheep: Could you please elaborate on the case where we don't get a message with a preview size? Perhaps in a comment?